### PR TITLE
US13 automation

### PR DESCRIPTION
**Pull request by yassirelkhemalli**

### CODE REVIEW

**Review by dilovar1992 — APPROVED** [review]
Any flaws? Before merge

File: src/test/java/com/fleet_management/step_definitions/US10_CalendarEvents_StepDef.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import org.junit.Assert;
+
+public class US10_CalendarEvents_StepDef {
+
+
+    US10_CalendarEventsPage us10_CalendarEventsPage = new US10_CalendarEventsPage();
+
+
+
+    @Then("user click to the Create Calendar event button")
```
Comment (ersuamine) [inline src/test/java/com/fleet_management/step_definitions/US10_CalendarEvents_StepDef.java:17]: This definition is same as 
US08 step definition, line 14.
This i duplicate

File: src/test/java/com/fleet_management/step_definitions/US13_VehicleCosts_StepDef.java [hunk 2]
```diff
@@ -43,5 +44,12 @@ public void userShouldBeAbelToCheckTheFirstCheckboxToSelectAllVehicleCosts() {
 
 
 
+    }
+
+    @And("user click to the next page button")
+    public void userClickToTheNextPageButton() {
+        us13_vehicleCostsPage.nextPageButton.click();
+        BrowserUtils.waitFor(3);
```
Comment (ersuamine) [inline src/test/java/com/fleet_management/step_definitions/US13_VehicleCosts_StepDef.java:52]: can you utilize an explicit wait method

File: src/test/resources/features/US10_CalendarEventsPage.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Feature: I want to type into the Description field when I create a calendar event
+
+  Background: User is already in the log in page
+    Given the user is on the login page
+
+  @AC1
+  Scenario Outline: Verify that users can type messages in to the Description field on the Calendar Events page
+    Given the user logged in as "<userType>"
+    And user clicks "Calendar Events" from "Activities" tab
+    Then user click to the Create Calendar event button
```
Comment (ersuamine) [inline src/test/resources/features/US10_CalendarEventsPage.feature:11]: You can copy the same step from US08 feature file, line 11

File: src/test/resources/features/US13_VehicleCostsPage.feature [hunk 4]
```diff
@@ -26,7 +26,11 @@ Feature: manage filters on the Vehicle Costs page
     Given the user logged in as "<userType>"
     And user clicks "Vehicle Costs" from "Fleet" tab
     Then user should be abel to check the first checkbox to select All Vehicle Costs
```
Comment (ersuamine) [inline src/test/resources/features/US13_VehicleCostsPage.feature:28]: there's a type in the word abel -> able

**Review by ersuamine — CHANGES_REQUESTED** [review]

**Pull request merged**